tests/test_tomography.py

```diff
@@ -167,3 +172,26 @@ def test_matrix_basis_n_decomp():
         matrix_rebuilt += mu[idx]*basis_matrices
 
     assert np.allclose(matrix, matrix_rebuilt)
+
+
+def test_qst_perf():
```

Comment:
I'm not sure this is a proper automated test we'd like to keep.
That's really good that you've compared computation times of before vs after your dev. However, now we don't perform unnecessary computations by design, thus reducing the computation time compared to the previous state of the tomography code. We know it for a fact because we improved the previous code. Previous auto tests ensure that the algorithm still works, I really think that's enough.